src/Microsoft.Dnx.Tooling/Restore/NuGet/NuGetv2Feed.cs

```diff
@@ -208,9 +207,10 @@ public PackageInfo BuildModel(string id, XElement element)
             return new PackageInfo
             {
                 // If 'Id' element exist, use its value as accurate package Id
-                // Otherwise, use the value of 'title' if it exist
-                // Use the given Id as final fallback if all elements above don't exist
-                Id = idElement?.Value ?? titleElement?.Value ?? id,
+                // Use the given Id as fallback if Id does not exist.
+                // Sometimes 'title' element is the same as Id and can be used as a fallback,
```

Comment:
I think you're right, if someone needs to know why `titleElement?.Value` isn't there anymore they can use git blame. Removing.